File: python/src/questao_1/main.py
from typing import List

def sumAll(numbers:List[int]):
    result = 0
    for number in numbers:
        result += number
    
    return result


def main():
    print("Digite 10 numeros inteiros: ")
    numbers = []
    while len(numbers) < 10:
        numbers.append(int(input()))
    
    print(sumAll(numbers))




if __name__ == '__main__':
    main()

File: java/src/questao_1/Main.java
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class Main {
    public static int sumAll(List<Integer> numbers) {
        int result = 0;
        for (int number : numbers) {
            result += number;
        }
        return result;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        List<Integer> numbers = new ArrayList<>();

        System.out.println("Digite 10 numeros inteiros: ");
        while (numbers.size() < 10) {
            numbers.add(scanner.nextInt());
        }
        scanner.close();

        System.out.println(sumAll(numbers));
    }
}
